docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[tune-for-indexing-speed]]
+== Tune for indexing speed
```

Comment:
Yes but some things are quick and/or so important you need to think about them first. Like bulk and refresh interval. They both help *so* much that it doesn't really matter if it is iops or cpu. I think.